<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroundDialog</class>
 <widget class="QDialog" name="GroundDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>482</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <widget class="QToolButton" name="toolButton_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/Icon/res/floor_4.png</normaloff>:/Icon/res/floor_4.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QToolButton" name="toolButton_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/Icon/res/floor_1.png</normaloff>:/Icon/res/floor_1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="toolButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/Icon/res/floor_3.png</normaloff>:/Icon/res/floor_3.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="toolButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/Icon/res/floor_2.png</normaloff>:/Icon/res/floor_2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QToolButton" name="toolButton_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/Icon/res/floor_5.png</normaloff>:/Icon/res/floor_5.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="toolButton_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/Icon/res/floor_6.png</normaloff>:/Icon/res/floor_6.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="toolButton_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/Icon/res/floor_7.png</normaloff>:/Icon/res/floor_7.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="toolButton_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/Icon/res/floor_8.png</normaloff>:/Icon/res/floor_8.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QToolButton" name="toolButton_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/Icon/res/floor_1_flip.png</normaloff>:/Icon/res/floor_1_flip.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QToolButton" name="toolButton_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/Icon/res/floor_3_flip.png</normaloff>:/Icon/res/floor_3_flip.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QToolButton" name="toolButton_11">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/Icon/res/floor_6_flip.png</normaloff>:/Icon/res/floor_6_flip.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="toolButton_12">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/Icon/res/floor_7_flip.png</normaloff>:/Icon/res/floor_7_flip.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="toolButton_13">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/Icon/res/floor_5_flip.png</normaloff>:/Icon/res/floor_5_flip.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
